sale_purchase_requisition/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2023-2023 Moduon - Andrea Cattalani
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl
+{
+    "name": "Sale purchase requisition",
+    "summary": "Connect your quotations with a purchase agreement",
+    "version": "14.0.1.0.0",
+    "category": "Sale",
+    "website": "https://github.com/OCA/sale-workflow",
+    "author": "Moduon, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": [
+        "sale_management",
+        "purchase_requisition",
+        "purchase",
+        "purchase_discount",
```

Comment:
`purchase` dependency not needed because `purchase_requisition` has the dependency
`purchase_discount`, why this dependency?